Gruntfile.js

```diff
@@ -41,7 +41,7 @@ module.exports = function(grunt) {
 			cleanup_js: ['dist/js/*.*', '!dist/js/app.*'],
 			cleanup_css: ['dist/css/*.css', '!dist/css/app.*.css']
 		},
-		jade: {
+		pug: {
```

Comment:
I don't get it: [gruntfile](https://github.com/ethereum/ethereum-org/blob/master/Gruntfile.js) file is already converted from jade to pug, why is it trying again? Are you sure you were synced when you did your pr?